### PR TITLE
Update new design of account menu

### LINKED ISSUE #744: Redesign account menu
https://www.figma.com/design/CWz4DatnJ7Qhl3JycJhbFY/TDrive---MUI3?node-id=2566-175&node-type=frame&t=1QqNSEeubhw4O2hc-0

image

Change icon and margins according to design

Make it possible to select an email

### PR DESCRIPTION
**Pull request by lethemanh**
Apply new design of account menu

## Description
Change account menu style by this new design https://www.figma.com/design/CWz4DatnJ7Qhl3JycJhbFY/TDrive---MUI3?node-id=2566-175&node-type=frame&t=1QqNSEeubhw4O2hc-0

## Related Issue
#744 

## Screenshots (if appropriate):
CleanShot 2024-12-11 at 15 19 40@2x
CleanShot 2024-12-11 at 15 19 23@2x

### CODE REVIEW

File: tdrive/frontend/src/app/atoms/icons-agnostic/index.tsx [hunk 1]
```diff
@@ -21,6 +21,9 @@ import { ReactComponent as StatusCheckDouble } from './assets/status-check-doubl
 import { ReactComponent as StatusCheck } from './assets/status-check.svg';
 import { ReactComponent as Users } from './assets/users.svg';
 import { ReactComponent as Sort } from './assets/sort.svg';
+import { ReactComponent as Envelop } from './assets/envelop.svg';
```
Comment (shepilov) [inline tdrive/frontend/src/app/atoms/icons-agnostic/index.tsx:24]: could you please take a look at the icons that we already have in the project and use them.

Probably it's a https://heroicons.com/

Comment (shepilov) [reply tdrive/frontend/src/app/atoms/icons-agnostic/index.tsx:24]: you can also find them here https://drive.twake.app/storybook/index.html?path=/story/atoms-avatar--default

Comment (lethemanh) [reply tdrive/frontend/src/app/atoms/icons-agnostic/index.tsx:24]: I've replaced with existed icons

Comment (ericlinagora) [reply tdrive/frontend/src/app/atoms/icons-agnostic/index.tsx:24]: > you can also find them here https://drive.twake.app/storybook/index.html?path=/story/atoms-avatar--default

is this why the frontend docker build takes 40m ??

**Comment by ericlinagora** [comment]
In the figma, the account dropdown is left aligned, obviously this is not possible, so based on the app grid, i think the intent of the artist is to align the right edges of the avatar and the menu

**Comment by lethemanh** [comment]
> In the figma, the account dropdown is left aligned, obviously this is not possible, so based on the app grid, i think the intent of the artist is to align the right edges of the avatar and the menu

Yes, it properly, I did not notice about that

**Comment by lethemanh** [comment]
> In the figma, the account dropdown is left aligned, obviously this is not possible, so based on the app grid, i think the intent of the artist is to align the right edges of the avatar and the menu

I think I will leave this because it requires to change a lot in Menu component to adjust this small update

**Review by ericlinagora — APPROVED** [review]

**Pull request merged**